src/main/java/org/getopt/luke/plugins/FsDirectory.java

```diff
@@ -157,6 +158,11 @@ public void renameFile(String from, String to) throws IOException {
     fs.rename(new Path(directory, from), target);
   }
 
+  @Override
+  public void syncMetaData() throws IOException {
+
```

Comment:
@jpbempel thanks for the PR. Do you know of a hadoop method to syncMetaData or does it even make sense on hdfs?